.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
       sudo: required
       env:
         - TYPE=crossdock
-        - DOCKER_COMPOSE_VERSION=1.8.0
+        - DOCKER_COMPOSE_VERSION=2.1.0
```

Comment:
`2.1.0` is not a valid version - looks like you want `1.10.0` instead.

https://github.com/docker/compose/releases